pontoon/settings/base.py

```diff
@@ -753,22 +753,69 @@ def _default_from_email():
 )
 
 # Logging
+# Get environment variables
+LOG_TO_FILE = os.getenv("LOG_TO_FILE", "False") == "True"
+
+# Ensure the logs directory exists
+if LOG_TO_FILE:
+    log_dir = path("logs")
+    os.makedirs(log_dir, exist_ok=True)
+
+# Define handlers
+console_handler = {
+    "class": "logging.StreamHandler",
+    "formatter": "verbose",
+}
+
+django_file_handler = {
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": path("logs", "django_debug.log"),
+    "maxBytes": 1024 * 1024 * 2,  # 2 MB
+    "backupCount": 3,
+    "formatter": "verbose",
+}
+
+pontoon_file_handler = {
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": path("logs", "pontoon_debug.log"),
+    "maxBytes": 1024 * 1024 * 2,  # 2 MB
+    "backupCount": 3,
+    "formatter": "verbose",
+}
+
+# Define logging configuration
 LOGGING = {
     "version": 1,
     "disable_existing_loggers": False,
-    "handlers": {"console": {"class": "logging.StreamHandler"}},
+    "handlers": {
+        "console": console_handler,
+    },
     "formatters": {
-        "verbose": {"format": "[%(levelname)s:%(name)s] %(asctime)s %(message)s"},
+        "verbose": {
+            "format": "[%(levelname)s:%(name)s] %(asctime)s %(message)s",
+        },
     },
     "loggers": {
-        "django": {"handlers": ["console"]},
+        "django": {
+            "handlers": ["console"],
+            "level": os.environ.get("DJANGO_LOG_LEVEL", "DEBUG" if DEBUG else "INFO"),
+            "propagate": True,
+        },
         "pontoon": {
             "handlers": ["console"],
             "level": os.environ.get("DJANGO_LOG_LEVEL", "DEBUG" if DEBUG else "INFO"),
+            "propagate": True,
```

Comment:
`propagate` defaults to `True`:
https://docs.djangoproject.com/en/5.0/howto/logging/#using-logger-hierarchies-and-propagation

So what's the motivation for this change?